.ci/deps.apt-new.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
unused file; should be deleted.  Also the two files above changing only permissions dont need to be in this PR

Reply:
this was part of the Circle CI v1.0 system, which doesnt work now, and needs to be replaced with Circle CI v2.0 (https://github.com/coala/coala-bears/issues/2298), or deleted.

Reply:
Thoughts about Circle CI at https://github.com/coala/coala/issues/5945

Reply:
This was carried over from https://github.com/coala/coala-bears/pull/1386/files#diff-6c501d251c7c1c762440c0cfc399a630

That version of the PR had slightly better support for R , and for different Debian releases, but I dont think there is much to be gained by adding more Debian precise jobs - too old.

Travis does have Bionic images available, but not currently advertised/supported.
https://travis-ci.community/t/ubuntu-18-04-1-lts-bionic-beaver/1270/21